content/blog/2024/202401-review.rst

```diff
@@ -0,0 +1,68 @@
+================================================================
+sciwork January 2024 review: weekly meetup and Hsinchu scisprint
+================================================================
+
+:date: 2024-03-04
+:author: Ming-Jie Kuo
+:category: blog
+
+Weekly Meetup
+-------------
+In the January meetup, Yung-Yu introduced the CESE method, which is utilized in
+the modmesh project to solve the fluid dynamics problem. This provided modmesh
+members with a deeper understanding of using numerical methods to solve the partial
+differential equations. Meanwhile, Ming-Jie delved into researching the CESE method.
+In addition, Kuan-Wei Chiu and Chun-Shih, former students of the nsd course
```

Comment:
Please do not make contributor background explicit, unless you are asked to do so, or it is necessary.  It's to respect other's privacy.

Reply:
I removed the contributors' background.